README.md

```diff
@@ -39,8 +39,9 @@ Building
 Before even building, Alure requires the OpenAL development files installed, for example, through 
 Creative's OpenAL SDK (available from openal.org) or from OpenAL Soft. 
 
-Additional 
-dependencies are required for native playback of audio files:
+These following dependencies are only needed to automatically support the formats they handle; a plain wav file 
+decoder is built-in to automatically handle basic wav files, and app-provided custom decoders are allowed, 
+so they aren't strictly required to handle audio files in general. 
```

Comment:
This is pretty much just repeating what's said earlier under Features. It might be better to remove everything after the semicolon and turn the following list into links for where the official sites and downloads are.